### ISSUE TITLE
displayBanner is not in the DB anymore

### ISSUE THREAD

**Issue by dali-rajab** [PrestaShop/PrestaShop#22864]
#### Describe the bug

I can't find the "displayBanner" hook in this list, and thats for all modules (& widgets)

![modulespos](https://user-images.githubusercontent.com/39015823/104818964-0d01ec80-582b-11eb-85bd-fab754cc3ab8.jpg)

#### Hints

this function (which fetch the ${_PREFIX_}hook table) dont get the "displayBanner" hook :
```php
public static function getHooks($position = false, $only_display_hooks = false)
```
Its because the DB table  **${_PREFIX_}hook** dont contain the "displayBanner" hook.

I want to mention also that "displayBanner" hook is still here in the classic theme :
https://github.com/PrestaShop/PrestaShop/blob/d185234b517cd4d9bbdee202be6fcc108d02d0e2/themes/classic/templates/_partials/header.tpl#L27

#### Additional information

* PrestaShop version: 1.7.7.0

**Comment by hibatallahAouadni** [comment]
Hello @dali-rajab 

Thanks for your issue. Yes, this hook hasn't been registered yet, it's missing in the hook.xml
But, it's already fixed for 1.7.8.x

This issue is a duplicate of #17606. 
To be informed when it's released, please subscribe to the issue mentioned above.

Thanks!

**Comment by hibatallahAouadni** [comment]
Duplicate of #17606
